### PR TITLE
docs: Update documentation with some consistency

### PR DESCRIPTION
**Pull request by mamoodi**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**
No it doesn't resolve any open issues. This is for consistency.
* Remove unnecessary words and sections
* Update troubleshooting docs for better readability 

**Give a brief summary of what the PR does, explaining any non-trivial design decisions**

**Other references**

### CODE REVIEW

**Review by enyst — APPROVED** [review]

File: docs/modules/usage/custom_sandbox_guide.md [hunk 1]
```diff
@@ -34,9 +24,9 @@ Next build your docker image with the name of your choice, for example "custom_i
 
 This will produce a new image called ```custom_image``` that will be available in Docker Engine. 
 
-> Note that in the configuration described in this document, OpenDevin will run as user "opendevin" inside the sandbox and thus all packages installed via the docker file should be available to all users on the system, not just root
+> Note that in the configuration described in this document, OpenDevin will run as user "opendevin" inside the sandbox and thus all packages installed via the docker file should be available to all users on the system, not just root.
 > 
-> Installing with apt-get above installs node for all users 
+> Installing with apt-get above installs node for all users .
```
Comment (li-boxuan) [inline docs/modules/usage/custom_sandbox_guide.md:29]:
```suggestion
> Installing with apt-get above installs node for all users.
```

File: docs/modules/usage/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -19,11 +19,19 @@ open a *PR* to add details to this file.
 If you're running on Windows and having trouble, check out our [guide for Windows (WSL) users](troubleshooting/windows).
 :::
 
-## Unable to connect to Docker
+## Common Issues
+
+* [Unable to connect to Docker](#unable-to-connect-to-docker)
+* [Unable to connect to SSH box](#unable-to-connect-to-ssh-box)
+* [404 Resource not found](#404-resource-not-found)
+* [make-build-getting-stuck-on-package-installations](#make-build-getting-stuck-on-package-installations)
```
Comment (li-boxuan) [inline docs/modules/usage/troubleshooting/troubleshooting.md:27]:
```suggestion
* [`make build` getting stuck on package installations](#make-build-getting-stuck-on-package-installations)
```

File: docs/modules/usage/troubleshooting/troubleshooting.md [hunk 3]
```diff
@@ -19,11 +19,19 @@ open a *PR* to add details to this file.
...
+* [sessions-are-not-restored](#sessions-are-not-restored)
```
Comment (li-boxuan) [inline docs/modules/usage/troubleshooting/troubleshooting.md:28]:
```suggestion
* [Sessions are not restored](#sessions-are-not-restored)
```

**Review by li-boxuan — APPROVED** [review]

File: docs/modules/usage/troubleshooting/troubleshooting.md (hunk 3 above)
Comment (mamoodi) [reply docs/modules/usage/troubleshooting/troubleshooting.md]: Good catches!!! Thank you!

**Pull request merged**